bundle/builder.go

```diff
@@ -0,0 +1,72 @@
+package bundle
+
+import (
+	"bytes"
+	"context"
+	"io/fs"
+	"testing/fstest"
+
+	layerfs "github.com/dschmidt/go-layerfs"
+	opaBundle "github.com/open-policy-agent/opa/bundle"
+	opaCompile "github.com/open-policy-agent/opa/compile"
+)
+
+type Bundle struct {
+	fs fs.FS
+}
+
+func Build(bundle Bundle) ([]byte, error) {
```

Comment:
I wonder how we can test it.
As a first step, just making sure that we don't get errors by building `CISKubernetesBundle` and `CISEksBundle` could be nice.

Reply:
Sounds good, though I think Amir was aiming for something simpler like a basic unit-test.

Reply:
Yes, a unit test can do it